<?xml version="1.0" encoding="UTF-8"?><diffreport><diff><table border="0" cellpadding="1" cellspacing="0" width="100%"><tr><td><h3 class="ctxid">class:ExceptionInInitializerError [NONE]</h3></td></tr><tr><td><table border="0" cellpadding="5" cellspacing="1" bgcolor="black" width="100%"><tr valign="top"><td bgcolor="#FFFFFF" colspan="1"><div class="description"> <a start_ctx_id="class:ExceptionInInitializerError"/><ul class="blockList"><li class="blockList"><dl> <dt>All Implemented Interfaces:  </dt><dd> <a href="../../java/io/Serializable.html" title="interface in java.io">Serializable</a> </dd> </dl> <hr/> <br/><pre>public class <span class="typeNameLabel">ExceptionInInitializerError</span>
extends <a href="../../java/lang/LinkageError.html" title="class in java.lang">LinkageError</a></pre><div class="block">Signals that an unexpected exception has occurred in a static initializer. An <code>ExceptionInInitializerError</code> is thrown to indicate that an exception occurred during evaluation of a static initializer or the initializer for a static variable.<p>As of release 1.4, this exception has been retrofitted to conform to the general purpose exception-chaining mechanism. The "saved throwable object" that may be provided at construction time and accessed via the <a href="../../java/lang/ExceptionInInitializerError.html#getException--"><code>getException()</code></a> method is now known as the <i>cause</i>, and may be accessed via the <a href="../../java/lang/Throwable.html#getCause--"><code>Throwable.getCause()</code></a> method, as well as the aforementioned "legacy method."                                                      </p>                                 </div><dl> <dt> <span class="simpleTagLabel">Since:</span>   </dt><dd>JDK1.1</dd> <dt> <span class="seeLabel">See Also: </span>   </dt><dd> <a href="../../serialized-form.html#java.lang.ExceptionInInitializerError">Serialized Form </a> </dd>  </dl>  </li></ul> </div> <a end_ctx_id="class:ExceptionInInitializerError"/> </td></tr></table></td></tr></table><table border="0" cellpadding="1" cellspacing="0" width="100%"><tr><td><h3 class="ctxid">constructor:ExceptionInInitializerError-- [NONE]</h3></td></tr><tr><td><table border="0" cellpadding="5" cellspacing="1" bgcolor="black" width="100%"><tr valign="top"><td bgcolor="#FFFFFF" colspan="1"> <a start_ctx_id="constructor:ExceptionInInitializerError--"/><a name="ExceptionInInitializerError--"/><ul class="blockList"><li class="blockList"><h4>ExceptionInInitializerError</h4><pre>public ExceptionInInitializerError()</pre><div class="block">Constructs an <code>ExceptionInInitializerError</code> with <code>null</code> as its detail message string and with no saved throwable object. A detail message is a String that describes this particular exception.                          </div></li></ul> <a end_ctx_id="constructor:ExceptionInInitializerError--"/>  </td></tr></table></td></tr></table><table border="0" cellpadding="1" cellspacing="0" width="100%"><tr><td><h3 class="ctxid">constructor:ExceptionInInitializerError-java.lang.Throwable- [NONE]</h3></td></tr><tr><td><table border="0" cellpadding="5" cellspacing="1" bgcolor="black" width="100%"><tr valign="top"><td bgcolor="#FFFFFF" colspan="1"> <a start_ctx_id="constructor:ExceptionInInitializerError-java.lang.Throwable-"/><a name="ExceptionInInitializerError-java.lang.Throwable-"/><ul class="blockList"><li class="blockList"><h4>ExceptionInInitializerError</h4><pre>public ExceptionInInitializerError(<a href="../../java/lang/Throwable.html" title="class in java.lang">Throwable</a> thrown)</pre><div class="block">Constructs a new <code>ExceptionInInitializerError</code> class by saving a reference to the <code>Throwable</code> object thrown for later retrieval by the <a href="../../java/lang/ExceptionInInitializerError.html#getException--"><code>getException()</code></a> method. The detail message string is set to <code>null</code>.                            </div><dl> <dt> <span class="paramLabel">Parameters:</span>   </dt><dd> <code>thrown</code> - The exception thrown     </dd> </dl></li></ul> <a end_ctx_id="constructor:ExceptionInInitializerError-java.lang.Throwable-"/>  </td></tr></table></td></tr></table><table border="0" cellpadding="1" cellspacing="0" width="100%"><tr><td><h3 class="ctxid">constructor:ExceptionInInitializerError-java.lang.String- [NONE]</h3></td></tr><tr><td><table border="0" cellpadding="5" cellspacing="1" bgcolor="black" width="100%"><tr valign="top"><td bgcolor="#FFFFFF" colspan="1"> <a start_ctx_id="constructor:ExceptionInInitializerError-java.lang.String-"/><a name="ExceptionInInitializerError-java.lang.String-"/><ul class="blockListLast"><li class="blockList"><h4>ExceptionInInitializerError</h4><pre>public ExceptionInInitializerError(<a href="../../java/lang/String.html" title="class in java.lang">String</a> s)</pre><div class="block">Constructs an ExceptionInInitializerError with the specified detail message string. A detail message is a String that describes this particular exception. The detail message string is saved for later retrieval by the <a href="../../java/lang/Throwable.html#getMessage--"><code>Throwable.getMessage()</code></a> method. There is no saved throwable object.                                      </div><dl> <dt> <span class="paramLabel">Parameters:</span>   </dt><dd> <code>s</code> - the detail message     </dd> </dl></li></ul> <a end_ctx_id="constructor:ExceptionInInitializerError-java.lang.String-"/>  </td></tr></table></td></tr></table><table border="0" cellpadding="1" cellspacing="0" width="100%"><tr><td><h3 class="ctxid">method:getException-- [NONE]</h3></td></tr><tr><td><table border="0" cellpadding="5" cellspacing="1" bgcolor="black" width="100%"><tr valign="top"><td bgcolor="#FFFFFF" colspan="1"> <a start_ctx_id="method:getException--"/><a name="getException--"/><ul class="blockList"><li class="blockList"><h4>getException</h4><pre>public <a href="../../java/lang/Throwable.html" title="class in java.lang">Throwable</a> getException()</pre><div class="block">Returns the exception that occurred during a static initialization that caused this error to be created.<p>This method predates the general-purpose exception chaining facility. The <a href="../../java/lang/Throwable.html#getCause--"><code>Throwable.getCause()</code></a> method is now the preferred means of obtaining this information.                   </p>               </div><dl> <dt> <span class="returnLabel">Returns:</span>   </dt><dd>the saved throwable object of this <code>ExceptionInInitializerError</code>, or <code>null</code> if this <code>ExceptionInInitializerError</code> has no saved throwable object.                </dd> </dl></li></ul> <a end_ctx_id="method:getException--"/>  </td></tr></table></td></tr></table><table border="0" cellpadding="1" cellspacing="0" width="100%"><tr><td><h3 class="ctxid">method:getCause-- [NONE]</h3></td></tr><tr><td><table border="0" cellpadding="5" cellspacing="1" bgcolor="black" width="100%"><tr valign="top"><td bgcolor="#FFFFFF" colspan="1"> <a start_ctx_id="method:getCause--"/><a name="getCause--"/><ul class="blockListLast"><li class="blockList"><h4>getCause</h4><pre>public <a href="../../java/lang/Throwable.html" title="class in java.lang">Throwable</a> getCause()</pre><div class="block">Returns the cause of this error (the exception that occurred during a static initialization that caused this error to be created).                    </div><dl> <dt> <span class="overrideSpecifyLabel">Overrides:</span>   </dt><dd> <code><a href="../../java/lang/Throwable.html#getCause--">getCause</a></code> in class <code><a href="../../java/lang/Throwable.html" title="class in java.lang">Throwable</a></code>  </dd> <dt> <span class="returnLabel">Returns:</span>   </dt><dd>the cause of this error or <code>null</code> if the cause is nonexistent or unknown.             </dd> <dt> <span class="simpleTagLabel">Since:</span>   </dt><dd>1.4</dd>   </dl></li></ul> <a end_ctx_id="method:getCause--"/>  </td></tr></table></td></tr></table></diff></diffreport>